### ISSUE TITLE
Senior model Torch not learning

### ISSUE THREAD

**Issue by EricavanderSar** [FraunhoferIEE/CurriculumAgent#9]
The Senior model:  `Grid2OpCustomModelTorch()` in  `[senior_model_rllib.py`](https://github.com/FraunhoferIEE/curriculumagent/blob/master/curriculumagent/senior/rllib_execution/senior_model_rllib.py) is not able to learn anything. When creating training plots you will see the performance is not improving at all. 

Removing line 328 solved the problem for me, i.e. the softmax function should not be applied and the following lines should be removed:
```
        # Apply softmax to action
        action = self.softmax(action)
```
